modules/core/shared/src/main/scala/gem/Observation.scala

```diff
@@ -89,15 +94,30 @@ object Observation {
 
   }
 
-  /** Observation is a bitraversable functor. */
```

Comment:
I guess there is no such thing as a `Tritraverse`?

Reply:
It's possible to abstract over `NTraverse` but I don't think it's worth the effort. As long as none of the instances is implicit I think we're fine.